pkg/controller/atlasmap/atlasmap_controller.go

```diff
@@ -2,9 +2,15 @@ package atlasmap
 
```

Comment:
I'd prefer to see some testing added in when we introduce reconciler work & helper utils (even if just a couple simple ones - good practice). If none have been established for this project, preferably Ginkgo/Gomega to hold the line with OCP internal operators and other middleware.

@jamesnetherton any thoughts on framework preference?

Reply:
I am working on adding the test case to the existing.